### PR TITLE
AP_Soaring: add documentation for VAR dataflash log message

### PR DESCRIPTION
**Pull request by peterbarker**
I'm quite sure these could do with some improvement.  Still, better than what we currently have.

### CODE REVIEW

File: libraries/AP_Soaring/Variometer.cpp [hunk 1]
```diff
@@ -75,6 +75,21 @@ void Variometer::update(const float polar_K, const float polar_Cd0, const float
...
 
+// @LoggerMessage: VAR
+// @Vehicles: Plane
+// @Description: Variometer data
+// @Field: TimeUS: Time since system startup
+// @Field: aspd_raw: always zero
+// @Field: aspd_filt: filtered and constrained airspeed
+// @Field: alt: AHRS altitude
+// @Field: roll: AHRS roll
+// @Field: raw: still-air sinkrate
```
Comment (samuelctabor) [inline libraries/AP_Soaring/Variometer.cpp:86]: estimated air vertical speed

File: libraries/AP_Soaring/Variometer.cpp [hunk 2]
```diff
@@ -75,6 +75,21 @@ void Variometer::update(const float polar_K, const float polar_Cd0, const float
...
+// @Field: filt: filtered still-air sinkrate
```
Comment (samuelctabor) [inline libraries/AP_Soaring/Variometer.cpp:87]: low-pass filtered air vertical speed

File: libraries/AP_Soaring/Variometer.cpp [hunk 3]
```diff
@@ -75,6 +75,21 @@ void Variometer::update(const float polar_K, const float polar_Cd0, const float
...
+// @Field: cl: raw climb rate
+// @Field: fc: filtered climb rate
+// @Field: exs: expected thermalling sink
```
Comment (samuelctabor) [inline libraries/AP_Soaring/Variometer.cpp:90]: expected sink rate relative to air in thermalling turn

File: libraries/AP_Soaring/Variometer.cpp (hunk 1 above)
Comment (peterbarker) [reply libraries/AP_Soaring/Variometer.cpp]: Fixed

File: libraries/AP_Soaring/Variometer.cpp (hunk 2 above)
Comment (peterbarker) [reply libraries/AP_Soaring/Variometer.cpp]: Fixed

File: libraries/AP_Soaring/Variometer.cpp (hunk 3 above)
Comment (peterbarker) [reply libraries/AP_Soaring/Variometer.cpp]: Fixed

**Pull request merged**